driver.go

```diff
@@ -159,8 +165,8 @@ func connect(opts *Options) (*Conn, error) {
 
 	httpTransport, ok := transport.(*thrift.THttpClient)
 	if ok {
-		// TODO: currently masking as a python connector until additional user agents are white listed.
-		httpTransport.SetHeader("User-Agent", "godatabrickssqlconnector/0.9.0 (go-dbsql)")
+		userAgent := fmt.Sprintf("godatabrickssqlconnector/0.9.0 (%s)", opts.ClientId)
```

Comment:
Can you pull the version out as an option as well?

Reply:
I think Moe makes a good point below. I'll pull the version out as an option but will not allow end user to override the version in the URI used to open connection